src/main/java/com/dev/cinema/config/SecurityConfig.java

```diff
@@ -1,39 +1,70 @@
 package com.dev.cinema.config;
 
+import com.dev.cinema.model.Role;
+import com.dev.cinema.model.User;
+import com.dev.cinema.service.UserService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
 import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.userdetails.UserDetailsService;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
 
 @Configuration
 @EnableWebSecurity
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
+    private static final Logger LOGGER =
+            LogManager.getLogger(SecurityConfig.class);
+    private final UserDetailsService userDetailsService;
+    private final UserService userService;
+
+    public SecurityConfig(UserDetailsService userDetailsService, UserService userService) {
+        this.userDetailsService = userDetailsService;
+        this.userService = userService;
+    }
+
     @Autowired
     public void configureGlobal(AuthenticationManagerBuilder authentication) throws Exception {
+        LOGGER.info("try to create user");
+        User user = new User("admin@user.com", getEncoder().encode("admin"));
+        user.addRoles(Role.of("ADMIN"));
```

Comment:
```suggestion
        user.addRoles(Set.of(roleService.getByName("ADMIN")));
```
ось таке це приблизно повинно виглядати

Reply:
чому так?

Reply:
++